docs/Project-Configuration/Task_Traces.md

```diff
@@ -27,11 +27,12 @@ The following examples illustrate how to inject the trace/span IDs into the cont
 #### Python
 ```python
 from opentelemetry import trace, context
-from opentelemetry.trace import NonRecordingSpan, SpanContext
+from opentelemetry.trace import NonRecordingSpan, SpanContext, TraceFlags
 
 span_context = SpanContext(
     trace_id = int("${otel_trace_id}", 16),
-    span_id = int(("${otel_parent_id}", 16)
+    span_id = int(("${otel_parent_id}", 16),
+    trace_flags = TraceFlags(0x01)
```

Comment:
There is an `is_remote` flag that is required here. Could you give guidance on what the correct option should be?